2-mandatory/3-financial-times.js

```diff
@@ -23,15 +39,27 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    let arrNum = []; 
+    for (let i = 0; i < allArticleTitles.length; i++) {
```

Comment:
This is an interesting implementation :)
You could make this a little bit more efficient using the `break` keyword. Basically, in the inner loop - if you've already added the current word to the array, you can break out of the loop, and just carry on to the next word.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/break